be/src/script/script.cpp

```diff
@@ -119,6 +121,41 @@ static int64_t unix_seconds() {
     return UnixSeconds();
 }
 
+static std::string exec(const std::string& cmd) {
+    std::string ret;
+
+    FILE* fp = popen(cmd.c_str(), "r");
+    if (fp == NULL) {
+        ret = strings::Substitute("popen failed: $0 cmd: ", strerror(errno), cmd);
```

Comment:
```suggestion
        ret = strings::Substitute("open file failed: $0 cmd: $1", strerror(errno), cmd);
```

Reply:
it's `popen` system call to open a new process, not opening a file.